core/src/test/java/io/questdb/test/cairo/mv/IntListTest.java

```diff
@@ -1,302 +0,0 @@
-/*******************************************************************************
- *     ___                  _   ____  ____
```

Comment:
is this a copy of another tests?

Reply:
Yes, that was a copy of `io.questdb.test.std.IntListTest`